From d5b752675d63d4a75cfb6cfc4173309a404f6074 Mon Sep 17 00:00:00 2001
From: Ryan McCabe <rmccabe@redhat.com>
Date: Wed, 28 Nov 2012 12:59:15 -0500
Subject: [PATCH] luci: Add missing JS include

Add a missing javascript include to the services page that caused
changing the state of the "force unmount" checkbox to not update the
state of the nfsrestart checkbox.

Resolves: rhbz#822502

Signed-off-by: Ryan McCabe <rmccabe@redhat.com>
---
 luci/templates/service.html | 1 +
 1 file changed, 1 insertion(+)

diff --git a/luci/templates/service.html b/luci/templates/service.html
index ef76bea..93a3995 100644
--- a/luci/templates/service.html
+++ b/luci/templates/service.html
@@ -20,6 +20,7 @@
                cluster_status = tmpl_context.cluster.get_status()">
   <script type="text/javascript" src="/js/encoder.js"></script>
   <script type="text/javascript" src="/js/service.js"></script>
+  <script type="text/javascript" src="/js/resource.js"></script>
   <xi:include href="resource_list.html" />
   ${resource_container(cluster_data)}
 
-- 
1.7.11.7

